tests/manifests/test_tabular.py

```diff
@@ -244,3 +244,68 @@ def test_with_denormalized_data(tmp_path, rc):
       |   |   |   | country.name           |        |           | open
       |   |   |   | country.continent.name |        |           | open
     ''')
+
+
+def test_prop_array(rc):
+    rc = rc.fork({
+        'default_auth_client': 'default',
+    })
+    bootstrap_manifest(rc, '''
```

Comment:
Use `check` as in all other tests here. And for other array related tests, use `tests/dtypes/test_array.py`. You can use `tests/dtypes/geometry.py` as an example.